.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,14 +6,14 @@ A brief description of the change being made with this pull request.
 
 What inspired you to submit this pull request?
 
-### Review checklist
-
-- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
-- [ ] Feature or bugfix has tests
-- [ ] Git history is clean
-- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)
-- [ ] If PR adds a configuration option, it has been added to the configuration file.
-
 ### Additional Notes
 
 Anything else we should know when reviewing?
+
+### Review checklist (to be filled by reviewers)
+
+- [ ] PR title is written as a CHANGELOG entry [(see why)](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title)
+- [ ] PR has the `changelog/` and `integration/` labels attached
+- [ ] Feature or bugfix has tests
+- [ ] Git history [is clean](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#commit-messages)
```

Comment:
I agree, especially since the only time we don't squash+merge is if someone (like a contributor) explicitly asks for rebase+merge to preserve distinct changes in same PR. However, that use case should be discouraged because our release tooling assumes a PR corresponds to one changelog entry only.

Reply:
What about forbidding `rebase+merge` all together and let people do whatever they want with commit messages - or suggest a placeholder like i described above for the time the pr stays open? It is true that at the end we `squash` so only the PR description matter and this one we usually take care of it naturally.

Reply:
I think the main goal of the clean git history is to ease review by allowing to review each commit 1 by 1, it's not really a question of whether or not we want to squash.

Reply:
I'm a little late to this one 😄 I think having a clean git history before you ask for a review is helpful, but once you've asked for one its a bit easy for the history to become messy. (Especially for bigger PRs and/or new integration PRs) Is the expectation then to force push over things as reviews/iterations happen?